tests/integration/test_signal_crashes.py

```diff
@@ -821,7 +821,7 @@ def test_crash_apport_from_systemd_coredump(self, reader_mock: MagicMock) -> Non
 
             now = datetime.datetime.now(tz=datetime.timezone.utc)
             expected_report = apport.report.Report(
-                date=now.strftime("%a %b %d %H:%M:%S %Y")
+                date=now.strftime(f"%a %b {now.day:>2} %H:%M:%S %Y")
```

Comment:
suggestion: use width specifier as documented in strftime(3)
```suggestion
                date=now.strftime("%a %b %2d %H:%M:%S %Y")
```

As hinted at by the Python documentation, `time.strftime()` is a thin wrapper around the libc implementation, and glibc supports width specifiers.

Reply:
That doesn't work: `%d` already resolves to the date with `0` prefixed. But `%e` works (but is not mentioned in the Python documentation).